scripts/find_versions.py

```diff
@@ -33,7 +33,7 @@ class PlatformTriple(NamedTuple):
     arch: str
     platform: str
     environment: str | None
-    flavor: str | None
+    flavor: str
```

Comment:
Changing this to use `""` as the nominal placeholder instead of `None` was just to avoid type checkers complaining about the possible `None` value when invoking `flavor.startswith(...)`